### ISSUE TITLE
Conversation order wrong

### ISSUE THREAD

**Issue by luciendutoitit** [MicrosoftDocs/msteams-docs#3249]
Hello,

For the last few days, the conversation order in Teams on Windows has been wrong. New messages get placed above older messages making conversations hard to follow, and it also makes missing crucial messages easy to miss. We've attempted to reboot our systems, as well as uninstalling and reinstalling the application.

Thank you

**Comment by Jagadeesh-MSFT** [comment]
Hi @luciendutoitit, we are unable to repro your issue from our end , are you sill facing the issue , could you please share the repro steps ?

**Comment by luciendutoitit** [comment]
Thanks for the reply, and apologies for my delay in responding.

Aside from attaching a screenshot exhibiting the behavior, there is not much else to do to replicate the issue.

An example is I send a message to someone on Teams, but their response gets nested above my message. Please see image.
![wrong order](https://user-images.githubusercontent.com/83778130/117677765-bf641380-b17c-11eb-864d-47e8f1c8cfeb.JPG)

Thank you
